gulpfile.js

```diff
@@ -16,5 +16,9 @@ require('laravel-elixir-vue-2');
 elixir(mix => {
     mix.sass('app.scss')
        .webpack('app.js')
-       .copy('node_modules/bootstrap-sass/assets/fonts/bootstrap/','public/fonts/bootstrap');;
+       .scripts([
+           '../../../node_modules/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.js'
```

Comment:
I think we should minify the output, before merge this PR into master.
`npm run production`